.github/workflows/webpage.yml

```
@@ -1,8 +1,7 @@
 name: Repository webpage
 
 on:
-  push:
-    branches: ["main"]
+  release:
```

I assume this change is based on

- https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onevent_nametypes
- https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#release

If so, it seems the syntax requires specifying particular types of releases. I would prefer only stable, non-draft & non-preliminary releases like so:

```suggestion
  release:
      types: [released]
```

Source: https://docs.github.com/en/webhooks/webhook-events-and-payloads?actionType=released#release

---

Sure, we can do that. I have a feeling if you don't specify the release type, it'll run the workflow for all releases, but it probably does make sense to narrow it down a bit.